### DISCUSSION TITLE
Welcome to TransQuest Discussions!

### DISCUSSION THREAD

**Discussion by TharinduDR** [TharinduDR/TransQuest#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cwenhaw** [comment]
Hi Tharindu, 
I think the training loss (tr_loss) in train_model(..) for MonoTransQuestModel needs to be set to 0 at the start of each epoch. If not, it will just keep accumulating over epochs. Thanks!
